pgql-lang/src/main/java/oracle/pgql/lang/Pgql.java

```diff
@@ -327,9 +327,8 @@ private PgqlVersion getPgqlVersion(IStrategoTerm ast, PgqlStatement statement) {
     }
 
     if (statement.getStatementType() == StatementType.CREATE_PROPERTY_GRAPH
-        || statement.getStatementType() == StatementType.DROP_PROPERTY_GRAPH
-        || statement.getStatementType() == StatementType.CREATE_EXTERNAL_SCHEMA
-        || statement.getStatementType() == StatementType.DROP_EXTERNAL_SCHEMA) {
+        || statement.getStatementType() == StatementType.CREATE_SUPER_PROPERTY_GRAPH
+        || statement.getStatementType() == StatementType.DROP_PROPERTY_GRAPH) {
       return PgqlVersion.V_1_3_OR_UP;
```

Comment:
What is this version for? Shouldn't it be a bigger number for CREATE_SUPER_PROPERTY_GRAPH?

Reply:
it's used to generate various error messages for things that are allowed in older PGQL versions but not in newer. Example:
 - `Use /* .. */ instead of // .. to introduce a comment` when using `// ..` in PGQL 1.3 and up.
 - `Double quoted variable references are only available in PGQL 1.3 and up`
 -  `Use n.prop ASC instead of ASC(n.prop)` (I believe in PGQL 1.0 and up)

there are only three "versions":   V_1_0, V_1_1_OR_V_1_2, V_1_3_OR_UP
this because we can't distinguish a SELECT query in PGQL 1.3 from a SELECT query in PGQL 1.4 or PGQL 1.5.
since the last version is V_1_3_OR_UP it includes all newer version ("or up"). A little ugly but once we deprecate+remove v.1.2 we can get rid of this versioning altogether.